xml/System.Security.Cryptography/AesCryptoServiceProvider.xml

```diff
@@ -625,6 +625,8 @@
 ## Remarks  
  The minimum size of the key is 128 bits, and the maximum size is 256 bits.  
   
+ Changing the KeySize value will reset the Key itself and cause a new random key to be generated. This will happen any time the setter of the KeySize property is invoked (including when invoked for the same value).  
```

Comment:
The technical aspects are accurate, invoking set_KeySize for AesCryptoServiceProvider (or, really, any SymmetricAlgorithm-derived type) will cause the Key property to get randomized to a new value of that size, even when doing `alg.KeySize = alg.KeySize`.

The sentence construction "will reset the Key itself" feels clunky to me, though.

Reply:
I can see that. Any suggestions?

Could omit the word "itself"? Or possibly just simplify it to "Changing the KeySize value will cause a new random key to be generated."